### DISCUSSION TITLE
Test du compteur d’énergie Sonoff Pow Ring POWCT - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#56] [Comments]
# Test du compteur d’énergie Sonoff Pow Ring POWCT - Haade.fr

Nouveau compteur d’énergie intelligent du marché, le Sonoff Pow Ring POWCT propose une mesure de l’énergie externe, un affichage digital, une antenne wifi déporté de qui exploiter au mieux sa consommation d’énergie

https://haade.fr/fr/blog/sonoff-pow-ring-powct-tests-compteur-energie

**Comment by GUMNRONIN** [comment]
bonjour pour moi le contact  sec  est pour piloter un contacteur jour/nuit ou  un relai  de delestage .

**Reply by haade-administrator** [reply]
oui on ne trouve pas beaucoup de modules contact sec, Sonoff en sort un courant décembre avec boitier mais toujours en wifi pas de Zigbee pour le moment

**Comment by mazkagaz** [comment]
Bonjour,
Est-ce que l'intégration HACS de ce compteur permet de différentier en temps réel l'état de consommation ou d'injection ?
J'aimerais le mettre directement après le disjoncteur général afin de piloter plusieurs routeurs solaires, mais j'ai lu dans de nombreux commentaires qu'il était impossible de différentier la puissance reçue de la puissance injectée dans home assistant : est-ce vrai ? Pour moi, si la distinction n'est pas possible, il ne me servira à rien.

**Reply by haade-administrator** [reply]
bonjour la distinction n'est pas possible

**Reply by mazkagaz** [reply]
Bonjour,

Désormais la distinction est possible : j'ai travaillé sur le firmware et j'ai obtenu pas mal d'informations qu'on n'avait pas avec le firmware d'origine.

J'ai tout mis sur [!mon github](https://github.com/mazkagaz/sonoff_powct_esphome)

Ça m'a pris du temps mais j'y suis arrivé :)

**Reply by haade-administrator** [reply]
Top le travail, merci pour la communauté HA, je vais tester et faire un article prochainement

**Reply by mazkagaz** [reply]
J'espère surtout que ce travail sera utile à d'autres.

J'ai surtout travaillé sur ce qui m'était utile, mais peut-être que d'autres besoins seront remontés par d'autres utilisateurs et peut-être que je serai en mesure d'y répondre.

C'est toujours un sentiment agréable lorsqu'on parvient à réaliser ce type de tâche. En tout cas, moi, ça m'a pris du temps mais je me suis régalé. Ça faisait longtemps qu'un projet ne m'avait pas tenu jusque tard dans la nuit pendant plusieurs jours :-)
